edx/cc2olx
# Black formatting

This PR reformats all python code (with exception for OLX constructing code) using `black` tool, and enables formatting checking in Travis.

**Testing instructions**:

1. Add many blank lines somewhere in `setup.py`.
2. Run `tox`. It should fail.

**Reviewers**
- [ ] @alangsto
